app/src/main/java/org/mozilla/fenix/components/Components.kt

```diff
@@ -20,33 +19,20 @@ import mozilla.components.feature.addons.update.DefaultAddonUpdater
 import mozilla.components.feature.autofill.AutofillConfiguration
 import mozilla.components.lib.publicsuffixlist.PublicSuffixList
 import mozilla.components.support.base.worker.Frequency
-import org.mozilla.fenix.BuildConfig
-import org.mozilla.fenix.Config
-import org.mozilla.fenix.FeatureFlags
-import org.mozilla.fenix.HomeActivity
-import org.mozilla.fenix.R
+import org.mozilla.fenix.*
```

Comment:
These `*` imports will get caught by the linter during CI. Please revert them. You can run our lint checks locally with
`./gradlew ktlint`
and
`./gradlew detekt`